framework/src/application/application.js

```diff
@@ -592,7 +587,7 @@ class Application {
 		const node = new Node({
 			channel: this.channel,
 			options: {
-				...this.config.app.node,
+				...this.config,
```

Comment:
This will pass `network`, `components` and all others as well, please strip those down to very specific that require from `Node`.